gcc/multiple_target.cc

```diff
@@ -298,6 +298,20 @@ create_target_clone (cgraph_node *node, bool definition, char *name,
   return new_node;
 }
 
+/* Check if AFMV option is provided in command */
+
+static bool afmv_enabled() {
+    
+    // extern bool global_afmv_enabled; // This should be defined where handles command-line options
+    bool global_afmv_enabled = true; /* Assuming AFMV option is provided for testing */
+	 
+    if (global_afmv_enabled) {
+        return true; /* Indicates afmv option provided */
+    }
+	
+    return false; /* No afmv option provided */
```

Comment:
You can return the global_afmv_enabled bool here instead of having two return statements.